src/java.desktop/macosx/native/libawt_lwawt/awt/CRobot.m

```diff
@@ -398,6 +423,12 @@ static inline CGKeyCode GetCGKeyCode(jint javaKeyCode)
     return [keyCodeMapping getOSXKeyCodeForJavaKey:javaKeyCode];
 }
 
+static inline int GetCGKeyMask(int cgKeyCode)
```

Comment:
```suggestion
static inline int GetCGKeyMask(int cgKeyCode) {
```

Reply:
I disagree: the common style throughout this file is to put the opening brace on the following line. A few lines above, `GetCGKeyCode` follows the same style.

Reply:
@aivanov-jdk Yeah, I wasn't sure of this - some methods follow Java-style of having the open brace on the same line and few others have it on the next line.

Reply:
Follow the style of the surrounding code so that added or modified lines don't stand out.

Reply:
Updated